### ISSUE TITLE
Choices Property Group - Screen Reader doesn't announce the expanded/collapsed state for the choice detail panel when focusing the Edit button

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-creator#3797]
Original issue: [T11831 - A11y bug : Screenreader is not announcing the state of 'Edit' button when it's expanded/collapsed]
(https://surveyjs.answerdesk.io/internal/ticket/details/T11831)
***
A screen reader (e.g., NVDA) doesn't announce the expanded / collapsed state for a choice's detail panel when focusing the Edit button:
![image](https://user-images.githubusercontent.com/22372972/214241097-39b320f3-ace1-4213-b5ee-97029f0ff612.png)
